etc/kayobe/ansible/requirements.yml

```diff
@@ -4,7 +4,8 @@ collections:
     version: 0.3.0
   - name: pulp.squeezer
     version: 0.0.11
+  - name: stackhpc.cephadm
+    version: 1.8.0
 roles:
   - name: vxlan
```

Comment:
Drop this line, otherwise the role will be renamed to `vxlan`. Move the `-` to the next line